.internal-ci/helm/fog-test-client/templates/fog-test-client-deployment.yaml

```diff
@@ -27,35 +27,35 @@ spec:
         runAsGroup: 1000
       containers:
       - name: fog-test-client
-        image: "{{ .Values.fogTestClient.image.repository }}:{{ .Values.fogTestClient.image.tag }}"
+        image: '{{ .Values.fogTestClient.image.repository }}:{{ .Values.fogTestClient.image.tag | default .Chart.AppVersion }}'
         imagePullPolicy: Always
         securityContext:
           capabilities:
             drop:
             - all
           readOnlyRootFilesystem: true
         env:
-        - name: FOG_VIEW
+        - name: MC_FOG_VIEW
           valueFrom:
             configMapKeyRef:
               name: {{ .Values.fogTestClientConfig.configMap.name }}
               key: FOG_VIEW
-        - name: FOG_LEDGER
+        - name: MC_FOG_LEDGER
           valueFrom:
             configMapKeyRef:
               name: {{ .Values.fogTestClientConfig.configMap.name }}
               key: FOG_LEDGER
-        - name: CONSENSUS_VALIDATORS
+        - name: MC_CONSENSUS_VALIDATORS
```

Comment:
Note: I didn't change the external config here, just the mapping from configmap to the runtime.